internal/passwordutil/password.go

```diff
@@ -0,0 +1,74 @@
+package passwordutil
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	nanoid "github.com/matoous/go-nanoid/v2"
+
+	"github.com/planetscale/cli/internal/cmdutil"
+	ps "github.com/planetscale/planetscale-go/planetscale"
+)
+
+type Options struct {
+	Organization string
+	Database     string
+	Branch       string
+	Role         cmdutil.PasswordRole
+	Name         string
+	TTL          time.Duration
+}
+
+type Password struct {
+	ps.DatabaseBranchPassword
+	cleanup func(context.Context) error
```

Comment:
Please let's used keyed fields. It breaks the existing struct constructions the moment you extend this struct.

Reply:
> Otherwise, I think what you're suggesting is it'd need to be like,

Yeap that's what I suggested. The previous version is called "struct with unkeyed fields", hence I suggested adding field keys. 

The reason keyed structs are better is they are maintainable and easier to read. As an example, if you add a new field to the `Password` struct, if you would use unkeyed fields, it would break the compilation, and you have to refactor all existing usages of the `Password` struct. Obviously here it's just a single place, but in larger codebases it's a lot of hassle in long term. With keyed fields, adding a new field doesn't break existing usages, and everything works as it is, and no further changes are required.

Reply:
Yeah, I understand the differences and made a conscious decision on my choice for the ergonomics that I felt were better. I made the change anyways as you preferred, I just disagree that it's better that way, but it's bike shedding. I also chose to embed the struct to not have another pointer dereference, but that's an extreme nit on my part. Just something tangibly different in the implementations.